Yoast/wordpress-seo
# Add canonical presenter with tests

## Summary

Adds a presenter for the canonical tag

This PR can be summarized in the following changelog entry:

* N/A

## Test instructions

This PR can be tested by following these steps:

* Visit a post.
* The canonical tag should be set the same as before.

## UI changes

## Documentation

## Quality assurance


Fixes #
